addons/l10n_eg_edi_eta/models/account_move.py

```diff
@@ -19,7 +20,18 @@ class AccountMove(models.Model):
     l10n_eg_qr_code = fields.Char(string='ETA QR Code', compute='_compute_eta_qr_code_str')
     l10n_eg_submission_number = fields.Char(string='Submission ID', compute='_compute_eta_response_data', store=True, copy=False)
     l10n_eg_uuid = fields.Char(string='Document UUID', compute='_compute_eta_response_data', store=True, copy=False)
-    l10n_eg_eta_json_doc_id = fields.Many2one('ir.attachment', copy=False)
+    l10n_eg_eta_json_doc_id = fields.Many2one(
+        comodel_name='ir.attachment',
```

Comment:
I'm not sure to understand. What's the point of this field ? And why do you still use a M2O with ir.attachment ? And why do you need to invalidate all the time, isn't the other field recomputed automatically with the dependence change ? :thinking: